### PR TITLE
Add Google Tasks

### LINKED ISSUE #6299: New icon for Google Tasks
**Brand Name:** Google Tasks
**Website:** Just available as App for Android and iOS
**Alexa rank:** _Since there is no website, there isn'ta rank ;)_

**Official resources for icon and color:**
Google do not provide an official source but here is an article about the newest logo:
https://9to5google.com/2018/03/30/google-tasks-redesign-logo-leak/

Should also be added to the Google Icon Review issue to track the Tasks logo too https://github.com/simple-icons/simple-icons/issues/4205

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
googletasks

**Issue:** closes #6299 

**Similarweb rank:** --. See https://www.similarweb.com/website/assistant.google.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from https://assistant.google.com/tasks by intercepting the loading page
Color: Taken from logo `#2684fc`

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5492,6 +5492,11 @@
             "hex": "246FDB",
             "source": "https://tagmanager.google.com/#/home"
         },
+        {
+            "title": "Google Tasks",
+            "hex": "2684FC",
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:5497]: Should we use the standard Google Blue here? 🤔 
```suggestion
            "hex": "4285F4",
```

File: _data/simple-icons.json [hunk 2]
```diff
@@ -5492,6 +5492,11 @@
...
+            "source": "https://assistant.google.com/tasks"
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:5498]: And do we need to add the usual Google guidelines?
```suggestion
            "source": "https://assistant.google.com/tasks",
            "guidelines": "https://about.google/brand-resource-center"
```

File: icons/googletasks.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Google Tasks</title><path d="M12.388 16.4a1.421 1.421 0 0 1-2.012 0l-4.164-4.164a1.139 1.139 0 0 1 0-1.609l1.774-1.774a1.139 1.139 0 0 1 1.609 0l1.756 1.756c-.003-.003 8.903-8.9 8.9-8.903a1.139 1.139 0 0 1 1.61 0l1.805 1.806a1.136 1.136 0 0 1 0 1.609l-11.278 11.28m6.37-4.762c.006.12.022.239.022.361a7.398 7.398 0 1 1-3.908-6.523l.125-.125 2.768-2.767A11.316 11.316 0 0 0 11.382.618C5.096.618 0 5.714 0 12s5.096 11.382 11.382 11.382S22.764 18.286 22.764 12c0-1.31-.222-2.568-.63-3.739l-3.376 3.378z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/googletasks.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Google Tasks</title><path d="M11.383.617C5.097.617 0 5.714 0 12c0 6.286 5.097 11.383 11.383 11.383 6.286 0 11.38-5.097 11.38-11.383a11.34 11.34 0 0 0-.878-4.389l-3.203 3.203c.062.387.1.782.1 1.186a7.398 7.398 0 1 1-7.4-7.398c1.499 0 2.889.448 4.054 1.214l2.857-2.857a11.325 11.325 0 0 0-6.91-2.342zm9.674.756c-.292 0-.583.112-.805.334-2.97 2.965-5.934 5.934-8.9 8.902L9.596 8.854a1.139 1.139 0 0 0-1.61 0l-1.775 1.773a1.139 1.139 0 0 0 0 1.61l4.166 4.163a1.421 1.421 0 0 0 2.012 0L23.666 5.121a1.136 1.136 0 0 0 0-1.61l-1.805-1.804a1.136 1.136 0 0 0-.804-.334z"/></svg>
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]

**Comment by PeterShaggyNoble** [comment]
Fixed some hairline differences in the cut-out. Don't know if this is the best treatment for us to use but, at the same time, I don't know what _is_ the correct treatment, other than maybe the circle overlapping the tick, so I think we should run with this and revisit if someone has another suggestion.

[Here's the original file](https://www.gstatic.com/companion/icon_assets/logo_tasks_2021_64dp.svg) for comparison.

File: _data/simple-icons.json (hunk 1 above)
Comment (adamrusted) [reply _data/simple-icons.json:5581]: They specifically use `#2684FC` here, so I'd say we stick with it (at least for now!)

**Review by adamrusted — APPROVED** [review]

**Pull request merged**